examples/height-stream/src/app.ts

```
@@ -0,0 +1,55 @@
+                    console.log(`Connected to height stream`);
+                    break;
+
+                case "Reconnecting":
+                    console.log(
+                        `Reconnecting to height stream in ${event.delayMillis}ms due to error: ${event.errorMsg}`,
+                    );
+                    break;
+
+                default:
+                    // Tells the typescript compiler that we have covered all possible event types
+                    const _exhaustiveCheck: never = event;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)